### PR TITLE
Wdp181202 3

### PR DESCRIPTION
**Pull request by ahna-code**
Problem: "Chodzi o elementy "Free shipping", etc. Pierwszy element ma ustawiony na sztywno stan "hover". Każda z korzyści powinna być linkiem i mieć hover".

Solution: Add links for h5 in content class, replace active selector with hover for feature-box and change style for h5 in content class.

### CODE REVIEW

File: src/sass/components/_feature-box.scss [hunk 1]
```diff
@@ -39,17 +39,22 @@
...
+      color: $text-color;
     }
 
     p {
       font-weight: 300;
     }
   }
 
-  &.active {
+    &:hover {
```
Comment (xkxd) [inline src/sass/components/_feature-box.scss]: this line has invalid indentation.

File: src/index.html [hunk 2]
```diff
@@ -93,7 +93,7 @@
               <i class="fas fa-truck"></i>
             </div>
             <div class="content">
-              <h5>Free shipping</h5>
+              <h5><a href="#">Free shipping</a></h5>
```
Comment (pawel-schmidt) [inline src/index.html]: Just suggestion: nested tags in one line are IMO less readable than broken into separate lines like this:
```
<h5>
  <a href="#">Free shipping</a>
</h5>
```

**Comment by Ma-Der** [comment]
It all seems OK.

**Comment by pawel-schmidt** [comment]
LGTM 👍

**Pull request merged**